src/commands/integrations/flows/test.ts

```diff
@@ -1,160 +1,382 @@
-      char: "t",
-      description: "Tail logs of the integration test run",
-      required: false,
+    "flow-url": Flags.string({
+      char: "u",
+      description: "Invocation URL of the flow to run.",
+    }),
+    "integration-id": Flags.string({
+      char: "i",
+      description: "ID of the integration containing the flow to test.",
     }),
     payload: Flags.string({
       char: "p",
-      description: "Optional JSON-formatted data payload to submit with the test",
-      required: false,
+      description: "Optional file containing a JSON-formatted data payload to run the flow with.",
+    }),
+    "payload-content-type": Flags.string({
+      description: "Optional content-type for the test payload.",
+      default: "application/json",
+    }),
+    sync: Flags.boolean({
+      description: "Forces the flow to run synchronously.",
     }),
-    contentType: Flags.string({
-      char: "c",
-      description: "Optional content-type for the test payload",
-      required: false,
+    "tail-results": Flags.boolean({
+      description: "Tail step results from the test execution until user interrupt or timeout.",
+    }),
+    "tail-logs": Flags.boolean({
+      description: "Tail logs from the test execution until user interrupt or timeout.",
+    }),
+    "cni-auto-end": Flags.boolean({
+      description:
+        "Automatically stop polling activity once an CNI flow execution completes. Some logs & results may not be returned this way. DOES NOT WORK FOR LOW-CODE FLOWS.",
+    }),
+    timeout: Flags.integer({
+      description:
+        "Optionally set a timeout (in seconds) to stop tail activity. Compatible with both low-code and CNI flows.",
+    }),
+    "result-file": Flags.string({
+      char: "r",
+      description:
+        "Optional file to append execution result data to. Results are saved as comma-separated values.",
     }),
   };
 
   async run() {
     const {
-      args: { flow },
-      flags: { tail, payload, contentType },
-    } = await this.parse(TestCommand);
-
-    const result = await gqlRequest({
-      document: gql`
-        mutation testIntegrationFlow(
-          $id: ID!
-          $payload: String
-          $contentType: String
-        ) {
-          testIntegrationFlow(
-            input: { id: $id, payload: $payload, contentType: $contentType }
-          ) {
-            testIntegrationFlowResult {
-              flow {
-                id
-              }
-              execution {
-                id
-              }
-            }
-            errors {
-              field
-              messages
-            }
-          }
+      flags: {
+        sync,
+        "flow-url": flowUrl,
+        "integration-id": integrationIdFlag,
+        payload: payloadFilePath,
+        "payload-content-type": contentType,
+        "tail-logs": tailLogs,
+        "tail-results": tailStepResults,
+        "cni-auto-end": autoEndPoll,
+        "result-file": resultFilePath,
```

Comment:
Were there any other flags (or arg) that could be added here to keep some existing functionality? If not, no biggie!

Reply:
`payload-content-type` is the new flag added to keep feature parity, otherwise everything else added is net new and we aren't missing anything from the old version.